boxes.lisp

```diff
@@ -537,3 +547,45 @@ N.B., Successive calls may return the same object."
 
 (defmethod baseline ((box sqrt-box))
   (baseline (sqrt-box-base box)))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; COLUMN BOXES ;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(deftype horizontal-alignment ()
+  "HORIZONTAL-ALIGNMENT specifies the alignments for COLUMN-BOX."
+  '(member :left :middle :right))
+
+(defclass column-box (box)
+  ((padding :initarg :padding
+            :type (integer 0)
+            :accessor column-box-padding)
+   (align :initarg :align
+          :accessor column-box-align
+          :type (horizontal-alignment)
```

Comment:
just say `horizontal-alignment` not `(horizontal-alignment)` (though the parenthesized version _is_ valid)